vulnerabilities/models.py

```diff
@@ -199,3 +200,25 @@ def make_data_source(self, batch_size: int, cutoff_date: datetime = None) -> Dat
 
     def __str__(self):
         return self.name
+
+
+class VulnerabilitySeverity(models.Model):
+
+    scoring_system_choices = [(system.identifier, system.name) for system in scoring_systems.values()]  # nopep8
```

Comment:
Why nopep8?
What about this?
```suggestion
    scoring_system_choices = tuple((ss.identifier, ss.name) for ss in scoring_systems.values())
```
BTW, I am also suggesting to use a tuple for something not supposed to change at runtime

Reply:
> Why nopep8?

Black ignores nopep8, I wanted this on a single line . 

re Tuple . 

Yes